app/src/test/java/org/simple/clinic/mobius/CustomFirstMatchers.kt

```diff
@@ -11,7 +11,7 @@ object CustomFirstMatchers {
 
     return object : TypeSafeDiagnosingMatcher<First<M, F>>() {
 
-      private val typeMatcher = Is.isA(type)
+      private val typeMatcher = Is.isA<Class<*>>(type)
```

Comment:
What would this line of code mean?

Reply:
I think it's basically just a hamcrest matcher to verify the passed type.